source/_integrations/buienradar.markdown

```diff
@@ -113,5 +113,7 @@ version of the name will be used as the name of the entity.
 With the default of "Buienradar loop" the entity name becomes
 `camera.buienradar_loop`.
 
+<div class='note'>
```

Comment:
I don't think it needs to be quoted or needs an alert box.

The most classical and elegant way would be marking that whole sentence italic.

Reply:
Done! not sure if you had in mind how I currently did it.